dm/dm/master/openapi.go

```diff
@@ -445,7 +445,7 @@ func (s *Server) DMAPIStartTask(c *gin.Context) {
 	for i := range subTaskConfigList {
 		subTaskConfigPList[i] = &subTaskConfigList[i]
 	}
-	if err = checker.CheckSyncConfigFunc(newCtx, subTaskConfigPList,
+	if _, err = checker.CheckSyncConfigFunc(newCtx, subTaskConfigPList,
```

Comment:
so user use openapi to start-task can not see the warnings ?, i thinks we need log theso these warnnings at leaset